lib/rubocop/cop/sorbet/rbi_versioning/gem_version_annotation_helper.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Sorbet
+      module GemVersionAnnotationHelper
+        VERSION_PREFIX = "# @version"
+
+        def gem_version_annotations
+          processed_source.comments.select do |comment|
+            gem_version_annotation?(comment)
+          end
+        end
+
+        private
+
+        def gem_version_annotation?(comment)
+          comment.text.start_with?(VERSION_PREFIX)
+        end
+
+        def gem_versions(comment)
+          comment.text.delete_prefix(VERSION_PREFIX).split(", ")
```

Comment:
Should we trim the blank spaces for the split string? Should we also use `/, ?/` to split? I'm not sure how strict we want to be in this cop?

Reply:
No I like this better, it makes everything more flexible.